benches/PERFORMANCE.json

```diff
@@ -0,0 +1,302 @@
+{
+  "d77ebe8481884bc3b2778c8083f1bf459e548e929edd87041beb14f6b868d35f": {
+    "benches": {
+      "big_values_data": {
+        "comment": "Loading a trace with huge values",
+        "stats": {
+          "air::farewell_step::outcome::from_success_result": {
```

Comment:
Suggest to delete `air::` prefix because it's common

Reply:
I calculate nested functions' common prefix and remove it in each node.